.github/workflows/build-test.yaml

```diff
@@ -23,3 +23,37 @@ jobs:
 
       - name: Build
         run: make build
+
+      - uses: actions/upload-artifact@v2
+        with:
+          name: coverage.out
```

Comment:
Question: What does this do? I thought jobs ran sequentially, so adding coverage here wouldn't do anything as there's no coverage.out yet?

Reply:
Coverage.out is produced by 'make test'

Reply:
You're completely right 🤦